### ISSUE TITLE
fix(formatter): on empty strings.numbers

### ISSUE THREAD

**Issue by rbalet** [ihym/ngx-timeago#71]
## Current behavior
`Argument of type 'NumberArray | undefined' is not assignable to parameter of type 'NumberArray'.
  Type 'undefined' is not assignable to type '[string, string, string, string, string, string, string, string, string, string]'`
![image](https://github.com/user-attachments/assets/c60fbb54-a463-4a43-b227-bf505a0c9c20)

## Expected behavior
Not having this error

## How do you think that we should fix this?
strings should not be empty, or at least a control should be done before hand to avoid crashing the code.  

Others:
Having a proper error response, like "timeAgo need x y" would be a nice to have

- [ ] Unkown word
![image](https://github.com/user-attachments/assets/8fec61d1-9b52-463e-b3d8-b254e8965914)

- [ ] Implicit 'any'
![image](https://github.com/user-attachments/assets/ac56863b-361f-4911-8d18-f684476c9761)
